plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveOnDataLake.java

```diff
@@ -1910,6 +1910,62 @@ public static Object[][] s3SelectFileFormats()
         };
     }
 
+    @Test
+    public void testTrinoDropSchemaCascade()
+    {
+        String schemaName = "test_drop_schema_cascade_" + randomNameSuffix();
+        String tableName = "test_table" + randomNameSuffix();
+        String trinoViewName = "test_trino_view" + randomNameSuffix();
+        String hiveViewName = "test_hive_view" + randomNameSuffix();
+        try {
+            assertUpdate("CREATE SCHEMA hive.%2$s WITH (location='s3a://%1$s/%2$s')".formatted(bucketName, schemaName));
+            assertUpdate("CREATE TABLE " + schemaName + "." + tableName + "(a INT)");
+            assertUpdate("CREATE VIEW " + schemaName + "." + trinoViewName + " AS SELECT 1 a");
+            hiveMinioDataLake.getHiveHadoop().runOnHive("CREATE VIEW " + schemaName + "." + hiveViewName + " AS SELECT * FROM " + schemaName + "." + tableName);
+
```

Comment:
Can we have a test case covering dropping a schema with external tables found in the same location as the schema?

Reply:
In this specific case dropping the managed table will cause dropping the data of the external table.

Could you please add another test where the external table's content is not actually dropped  and subsequently the `DROP SCHEMA CASCADE` query should fail?